tool/jt.rb

```diff
@@ -1740,7 +1741,7 @@ def checkout_or_update_graal_repo
     graal = Utilities.find_or_clone_repo('https://github.com/graalvm/graal.git')
 
     chdir(graal) do
-      raw_sh "git", "fetch", continue_on_failure: true
+      raw_sh "git", "fetch", "--all", continue_on_failure: true
```

Comment:
What does adding `--all` do here?
If you have a different remote than the upstream graal?